protostar/cli/common_arguments.py

```diff
@@ -0,0 +1,8 @@
+from .protostar_argument import ProtostarArgument
+
+lib_path_arg = ProtostarArgument(
```

Comment:
Constants should be all uppercase

Reply:
[use constant case](https://github.com/software-mansion/protostar/pull/1058/commits/009c1322db2dabc6584dd819891e0ddbe131be08)